src/Umbraco.Infrastructure/Migrations/Upgrade/UmbracoPlan.cs

```diff
@@ -16,6 +16,7 @@
 using Umbraco.Cms.Infrastructure.Migrations.Upgrade.V_9_1_0;
 using Umbraco.Cms.Infrastructure.Migrations.Upgrade.V_9_2_0;
 using Umbraco.Cms.Infrastructure.Migrations.Upgrade.V_9_3_0;
+using Umbraco.Cms.Infrastructure.Migrations.Upgrade.V_9_4_0;
```

Comment:
```suggestion
using Umbraco.Cms.Infrastructure.Migrations.Upgrade.V_10_0_0;
```

Reply:
This migration was added in 9.4.0 though, the commit introducing it into v10 is a cherrypick from v9